src/ui/components/CommandPalette/CommandButton.tsx

```diff
@@ -1,24 +1,32 @@
 import { formatKeyShortcut } from "devtools/client/debugger/src/utils/text";
-import React from "react";
+import React, { useEffect, useRef } from "react";
 import { connect, ConnectedProps } from "react-redux";
 import { actions } from "ui/actions";
 import { Command } from "./CommandPalette";
 
 type CommandButtonProps = PropsFromRedux & { command: Command; active: boolean };
 
 function CommandButton({ command, executeCommand, active }: CommandButtonProps) {
+  const buttonNode = useRef<HTMLButtonElement | null>(null);
   const { label, shortcut, key } = command;
   const onClick = () => {
     executeCommand(key);
   };
 
   const bgColors = active ? "bg-primaryAccent text-white" : "hover:bg-gray-200";
 
+  useEffect(() => {
+    if (active && buttonNode.current) {
+      buttonNode.current.scrollIntoView({ block: "nearest" });
+    }
```

Comment:
should we check that the button is not first visible.

Reply:
Looks like `nearest` keeps it from doing anything unpleasant.

https://user-images.githubusercontent.com/15959269/147703631-a40144bc-4306-4570-8c70-6995ed051fa3.mov